cohort_4/playbook/activities.md

```diff
@@ -58,6 +61,31 @@ first floor with the elevators.
 
 TODO
 
+## Demos
+
+### Presenting your demo
+
+Demos are the hackers' chance to show off what they have been putting so much
+time and effort into for the past day.
+
+- Whenever someone wants to present they can volunteer to (no set presentation
+  order)
+- Each team should start off with something they failed at
+  - Make sure to give context by explaining the basic overview of the app and
+    why this is a failure
+- After presenting something they failed at, they can show off what they made
+
+### Giving feedback
+
+To keep demos running smoothly, feedback is going to be written on sticky notes
+instead of shouted out after the demo.
```

Comment:
Perhaps add a quick explanation of what feedback constitutes and why we're doing it?